src/index.html

```diff
@@ -11,7 +11,152 @@
+            name="nameInput"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          How old are You?
+          <input
+            type="number"
+            name="ageInput"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input
+            type="date"
+            name="birthDate"
+          >
+        </label>
+        <label
+          class="label"
+          name="termAgreement">
+          I accept the term of the agreement
+          <input
+           type="checkbox"
+           required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form__field">
+        <legend>Registration</legend>
+        <label class="label">
+          E-mail:
+          <input
+            type="email"
+            name="emailInput"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="label">
+          Password:
+          <input
+            type="password"
+            name="passwordInput"
+            required
+            minlength="7"
+            maxlength="11"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form__field">
+        <legend>An interesting fact about you!</legend>
+        <label class="label">
+          Do you love cats?
+          <input
+            type="radio"
+            name="catLove"
+            value="yes"
+            id="choice1"
+          >
+          <label for="choice1">Yes</label>
+          <input
+            type="radio"
+            name="catLove"
+            value="no"
+            id="choice2"
+          >
+          <label for="choice2">No</label>
+        </label>
+        <label class="label">
+          What is your favorite color?
+          <input
+            type="color"
+            name ="favouriteColor"
+          >
+        </label>
+        <label class="label">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="bedTime"
+          >
+        </label>
+        <label class="label">
+          What are your favorite brand of cars?
+          <select
+            name="favouriteCar"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label class="label">
+          How do you rate our work?
+          <input
+            type="range"
+            name="workRange"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form__field">
+        <legend>Additional info:</legend>
+        <label class="label">
+          Comments:
+          <textarea
+            name="comment"
+            cols="15"
+            rows="2">
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="comment"
            cols="15"
            rows="2"
          ></textarea>
```

it should fix spaces in textarea field

Reply:
Committed changes, as requested. But npm test failed. Could you, please, explain why. Thank you